cmd/snap/cmd_snapshot.go

```diff
@@ -98,6 +101,10 @@ configuration data from the restore is not currently possible. This
 restriction may be lifted in the future.
 `)
 
+var longExportSnapshotHelp = i18n.G(`
+Export a snapshot into the target directory.
```

Comment:
Is this description accurate? Export takes a filename argument, not a directory, right?

Reply:
Updated